src/ddp/ethereum/index.ts

```diff
@@ -0,0 +1,47 @@
+import { ethers } from 'ethers'
+import { ZkWasmUtil } from '@hyperoracle/zkwasm-service-helper'
+import type { KeyofToArray } from '@murongg/utils/index'
+import { graph_abi } from '../../common/constants'
+import type { ProofParams } from '../../types'
+import { DataDestinationPlugin } from '../interface'
+
+export interface EthereumDDPGoParams {
+  signer: ethers.Wallet
+  gasLimit: number
+}
+
+export class EthereumDataDestinationPlugin extends DataDestinationPlugin<EthereumDDPGoParams> {
+  goParams: KeyofToArray<EthereumDDPGoParams> = ['signer']
+
+  async go(cleId: string, proofParams: ProofParams, goParams: EthereumDDPGoParams): Promise<void> {
+    const proof = ZkWasmUtil.bytesToBigIntArray(proofParams.aggregate_proof)
+    const instances = ZkWasmUtil.bytesToBigIntArray(proofParams.batch_instances)
+    const aux = ZkWasmUtil.bytesToBigIntArray(proofParams.aux)
+    const arg = ZkWasmUtil.bytesToBigIntArray(proofParams.instances)
+
+    try {
+      const graph = new ethers.Contract(cleId, graph_abi, goParams.signer)
+
+      const tx = await graph.trigger(proof, instances, aux, [arg], [0n], { gasLimit: goParams.gasLimit })
```

Comment:
Should print tx hash and the receipt information with enablelog option

Reply:
eslint prevent from console statement, will do this in future commit